src/reporters/apiary-reporter.coffee

```diff
@@ -222,18 +222,14 @@ class ApiaryReporter
       path: path
       method: method
       headers:
-        'User-Agent': "Dredd REST Reporter/" + packageData.version + " (" + system + ")"
+        'User-Agent': "Dredd Apiary Reporter/" + packageData.version + " (" + system + ")"
         'Content-Type': 'application/json'
         'Content-Length': Buffer.byteLength(postData, 'utf8')
 
     unless @configuration['apiToken'] == null
       options.headers['Authentication'] = 'Token ' + @configuration['apiToken']
 
-    if @verbose
-      info =
-        options: options
-        body: body
-      logger.log 'Rest Reporter Request:', JSON.stringify(info, null, 2)
+    logger.verbose('Apiary reporter request:', JSON.stringify({options: options, body}, null, 2))
```

Comment:
I don't understand your comment. But when I look at the change, `{options: options, body}` could be written as `{options, body}`.